src/states/node.rs

```diff
@@ -1593,6 +1616,72 @@ impl Node {
         result.with_value(Ok(()))
     }
 
+    fn handle_rt_req(&mut self,
+                     msg_id: MessageId,
+                     digest: sha256::Digest,
+                     src: Authority<XorName>,
+                     dst: Authority<XorName>)
+                     -> Evented<Result<(), RoutingError>> {
+        // TODO: Compare the hash and don't send anything if he requester is up-to-date.
+        let mut result = Evented::empty();
+        let sections = self.peer_mgr.pub_ids_by_section(self.full_id.public_id());
```

Comment:
That's true, but we use both values, so I'd have to add something like
```rust
fn sections_and_hash(&self) -> Result<(BTreeMap<...>, sha256::Digest), RoutingError>
```
to `Node`, I guess. I'm happy to do that, if you prefer it.